include/libtorrent/kademlia/get_item.hpp

```diff
@@ -53,13 +53,15 @@ class get_item : public find_data
 	// for immutable itms
 	get_item(node& dht_node
 		, node_id target
-		, data_callback const& dcallback);
+		, data_callback const& dcallback
+		, nodes_callback const& ncallback);
```

Comment:
I can't actually remember the exact semantics of these callbacks. When is this new callback called again? and would you mind adding a comment in the code?